src/Core/Domain/Order/QueryResult/OrderProductForViewing.php

```diff
@@ -234,6 +278,8 @@ public function getUnitPriceTaxInclRaw(): float
     }
 
     /**
+     * How much (money) has already been refunded for this product
+     *
      * @return string
      */
     public function getAmountRefund(): string
```

Comment:
```suggestion
    public function getAmountRefunded(): string
```
It will be easier to understand